recipes/boa/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "boa" %}
+{% set version = "0.0.1.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/boa/archive/{{ version }}.tar.gz
+  sha256: 534f89793dd8d7083b919d696f7d349d2c246df9b5f6763e7e7b9d34b16d1bd8
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - conda-mambabuild = boa.cli.mambabuild:main
+    - boa = boa.cli.boa:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pyyaml
+    - colorama
```

Comment:
Are pyyaml and colorama actually needed in host to install the package? I also think you don't need to list setuptools here.

Reply:
let's see!